roles/ocp4_workload_connectivity_link/tasks/workload.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Install Connectivity Link Operator
+  ansible.builtin.include_role:
+    name: install_operator
+  vars:
+    install_operator_action: install
+    install_operator_name: rhcl-operator
+    install_operator_namespace: openshift-operators
+    install_operator_manage_namespaces: []
+    install_operator_channel: "{{ ocp4_workload_connectivity_link_channel }}"
+    install_operator_catalog: redhat-operators
+    install_operator_automatic_install_plan_approval: "{{ ocp4_workload_connectivity_link_automatic_install_plan_approval | default(true) }}"
+    install_operator_starting_csv: "{{ ocp4_workload_connectivity_link_starting_csv }}"
+    install_operator_catalogsource_setup: "{{ ocp4_workload_connectivity_link_use_catalog_snapshot | default(false) }}"
+    install_operator_catalogsource_name: "{{ ocp4_workload_connectivity_link_catalogsource_name | default('') }}"
+    install_operator_catalogsource_namespace: openshift-operators
+    install_operator_catalogsource_image: "{{ ocp4_workload_connectivity_link_catalog_snapshot_image | default('') }}"
+    install_operator_catalogsource_image_tag: "{{ ocp4_workload_connectivity_link_catalog_snapshot_image_tag | default('') }}"
+
+- name: Create the Connectivity Link instance
+  kubernetes.core.k8s:
+    state: present
+    namespace: openshift-operators
+    definition: "{{ lookup('file', 'kuadrant.yaml') }}"
+
+- name: Wait until Kuadrant is installed
+  kubernetes.core.k8s_info:
+    api_version: kuadrant.io/v1beta1
+    kind: Kuadrant
+    name: kuadrant-instance
+    namespace: openshift-operators
+    wait: true
+    wait_condition:
+      reason: Ready
+      status: 'True'
+      type: Ready
+    wait_sleep: 10
```

Comment:
Can we just follow the normal pattern here? Is there a reason to deviate? Set it to 30 retries every 10 second or similar?